web/migrations/0055_rename_max_participants_to_max_attendees.py

```
@@ -0,0 +1,11 @@
+# Generated by Django 5.1.6 on 2025-04-15 21:45
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ("web", "0054_chapter_chapterapplication_chapterevent_and_more"),
+    ]
```

Ruff (0.8.2)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)